AK/Types.h

```diff
@@ -34,8 +34,6 @@ using f128 = long double;
 #    endif
 #endif
 
-#ifdef AK_OS_SERENITY
```

Comment:
This one is for sure not good. We can't be introducing the `size_t` definition from stddef.h and the ptrdiff/intptr types from stdint.h into the global namespace ourselves. We need to use AK in places that co-exist with system headers, such as in the Ladybird ports for Android/Qt/GTK/AppKit. Without multiple definitions of the same fundamental types..

Reply:
Resolved in favor of the next comment.